### PR TITLE
update packages

### PR DESCRIPTION
**Pull request by MP91**
# Description

## Checklist

* [ ] Vehicle App can be started with dapr run and is connecting to vehicle data broker
* [ ] Vehicle App can process MQTT messages and call the seat service
* [ ] Vehicle App can be deployed to local K3D and is running
* [ ] Created/updated tests, if necessary. Code Coverage percentage on new code shall be >= 70%.
* [ ] Extended the documentation in Velocitas repo
* [ ] Extended the documentation in README.md

* [ ] Devcontainer can be opened successfully
* [ ] Devcontainer can be opened successfully behind a corporate proxy
* [ ] Devcontainer can be re-built successfully

* [ ] Release workflow is passing

### CODE REVIEW

**Review by kse3hi — DISMISSED** [review]
LGTM

**Review by kse3hi — CHANGES_REQUESTED** [review]
Can you please and in app/requirements.in -> aiohttp >= 3.9.0 and re-compile dependencies...
It will fix 2 security issues and doesn't conflict with the DAPR version that we currently use.
https://github.com/dapr/python-sdk/blob/v1.11.0/setup.cfg

File: app/requirements.in [hunk 1]
```diff
@@ -16,3 +16,4 @@ grpcio==1.59.0
 protobuf==4.24.4
 dapr==1.11.0
 cloudevents==1.10.0
+aiohttp >= 3.9.0
```
Comment (kse3hi) [inline app/requirements.in:19]:
```suggestion
aiohttp == 3.9.0
```
To align with our "freeze version" strategy

**Review by kse3hi — CHANGES_REQUESTED** [review]

File: app/requirements.in (hunk 1 above)
Comment (kse3hi) [reply app/requirements.in]: No need to recompile - still same version was selected

Comment (MP91) [reply app/requirements.in]: already updated

**Review by kse3hi — APPROVED** [review]
LGTM

**Pull request merged**